Resources/public/js/views/tabs/ez-locationviewlocationstabview.js

```diff
@@ -106,6 +109,29 @@ YUI.add('ez-locationviewlocationstabview', function (Y) {
         },
 
         /**
+         * Tap event handler on the `Swap location` button. It fires the
+         * `swapLocation` event
+         *
+         * @method _swapLocation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _swapLocation: function (e) {
+            /**
+             * Fired when the user clicks on `Swap location` button
+             *
+             * @event swapLocation
+             * @param {eZ.Location} location the content to be swaped
+             * @param {Function} afterCreateCallback callback function that will be called after
```

Comment:
for this, as said above, this is more or less taken from another already existing methods... https://github.com/crevillo/PlatformUIBundle/blob/0337d024d0cc838a1401ee38ba3a43c0768cdb89/Resources/public/js/views/tabs/ez-locationviewlocationstabview.js#L96-L109

you mean param should be called `callback` instead of `afterSwapLoctionCallback`?